sysutils/helm/files/helm3.0

```diff
@@ -0,0 +1,2 @@
+bin/helm3.0
+-
```

Comment:
I'm curious as to why this line with `-` is here? Didn't just having line 1 in this file here work?

Reply:
Well, I was trying to understand the port select mechanism. I could not find documentation about that, so I tried to understand what it does by reading some source files and trial-and-error.
So with starting with helm V3, there is just one binary (helm), before that, there were two (helm and tiller). 
https://github.com/macports/macports-ports/blob/master/sysutils/helm/files/base references both helm and tiller. 
When a user has helm v2 installed and selected and is then installing some v3 helm and wants to select helm v3 via port select, then tiller (which is no part of helm v3) should not be available (i.e the symbolic link should be removed).
Looking at https://github.com/macports/macports-base/blob/master/src/macports1.0/macports.tcl#L4477, I would assume the correct indicator for exactly that case to be "-". (Also see the comment in Line 4478: `# The source is unavailable for this file`).
As this seemed consistent to me and it actually works, I have not investigated, if the second line could be omitted. Looking at the code, https://github.com/macports/macports-base/blob/master/src/macports1.0/macports.tcl#L4474 `set src [lindex $srcs $i]` will return an empty string (https://www.tcl.tk/man/tcl8.7/TclCmd/lindex.htm) for the target tiller and then enter the default case https://github.com/macports/macports-base/blob/master/src/macports1.0/macports.tcl#L4490 which might as a sideeffect remove the existing link and fail to create a new one. 
But as said, looking at the above mentioned comment https://github.com/macports/macports-base/blob/master/src/macports1.0/macports.tcl#L4478, I find the explicit approach of including a separate line for the tiller link to be removed more consistent.
And by the way, https://github.com/macports/macports-ports/blob/master/sysutils/helm/files/none also contains two lines with a "-".
I hope that comment is not too detailed, this is my first pull request..